lib/document/templates.js

```diff
@@ -123,8 +123,18 @@ const titlePage = (node) => {
   return ''
 }
 
+/**
+ * Generate an (hidden) outline otherwise Chrome won't generate "Dests" fields and we won't be able to generate a PDF outline.
+ */
+const outline = (node, baseConverter) => {
+  if (baseConverter) {
```

Comment:
I was a bit confused about where this value came from, but it all works fine apparently :)

Reply:
The baseConverter is basically `super` in other programming language.